endpointmanager/pkg/endpointmanager/postgresql/fhirendpointstore_test.go

```diff
@@ -95,6 +103,11 @@ func Test_PersistFHIREndpoint(t *testing.T) {
 
 	e1.OrganizationNames = []string{"Org 1", "Org 2"}
 	e1.NPIIDs = []string{"2", "3"}
+	var modvsr versionsoperatorparser.VersionsResponse
+	modvsr.Response = make(map[string]interface{})
+	modvsr.Response["default"] = "4.0"
+	modvsr.Response["versions"] = []string{"4.0","2.0"}
+	e1.VersionsResponse = modvsr
 	err = store.AddOrUpdateFHIREndpoint(ctx, e1)
```

Comment:
I think the VersionsResponse should be added to the AddFHIREndpoint function, even though I don't think there will be a case where the VersionsResponse would be added initially, since it would be useful to have it for tests like this